content/W2017/W17-event-techtalk-with-Monica-Beckwith.md

```diff
@@ -0,0 +1,18 @@
+Title: Techtalk with Monica Beckwith
+Date: 2017-03-14
+Category: Events
+Tags: social, talks
+Slug: tech-talk-with-Monica-Beckwith-w17
+Author: Fatima Taj
+Summary: Come join us at a tech talk with Monica Beckwith!
```

Comment:
Maybe something like "Monica Beckwith will be visiting campus to give a talk on her work in ...". You can have a look at https://github.com/wics-uw/website/blob/master/content/W2015/W15-event-hypatia.md for inspiration.